daemon/graphdriver/lcow/remotefs_pathdriver.go

```diff
@@ -0,0 +1,212 @@
+// +build windows
+
+package lcow
+
+import (
+	"errors"
+	"os"
+	pathpkg "path"
```

Comment:
The `path` variables in the function headers like `func (l *lcowfs) FromSlash(path string) string ` shadow the `path` package, so I aliased the package. I could also change the path variables in the function headers, but I wanted them to match the ones you see in `path/filepath`